azure-kusto-ingest/azure/kusto/ingest/exceptions.py

```diff
@@ -10,7 +10,10 @@ class KustoDuplicateMappingError(KustoClientError):
     """
 
     def __init__(self):
-        message = "Ingestion properties contains ingestion mapping and ingestion mapping reference."
+        message = (
```

Comment:
you meant to use 
```
"""
This is multi line string in python
"""
```

Reply:
Using this with several lines will result in a string that includes newlines, which - if I understand correctly - is not desirable in this case (printed out). What do you think?

Reply:
I think I expect to read code string, just as they will appear in real life. if you intend it to be a single line comment, just write it in a single line.

You will have to deal with linters, which we currently set to 120 chars. 
Do you think 120 chars applies to comments as well as code (hint: I do) ? If so, it should be multiline, other wise, use linting exclusions...

https://black.readthedocs.io/en/stable/the_black_code_style.html
> Black reformats entire files in place. It is not configurable. It doesn’t take previous formatting into account. It doesn’t reformat blocks that start with # fmt: off and end with # fmt: on